generators/profilegen/src/profile_generator.rb

```diff
@@ -69,12 +68,9 @@ def generate_modules
       @spec_includes << @module_require
       @class_name     = get_ruby_specific_resource_type_name(resource_provider)
       @class_names   << @class_name
-      if(!@individual_gem_profile)
```

Comment:
if this file will only be used for individual gem profile, should its name be updated to indicate this?